components/centraldashboard/OWNERS

```diff
@@ -1,10 +1,8 @@
 approvers:
-  - avdaredevil
   - elikatsis
   - kimwnasptd
-  - prodonjs
   - StefanoFioravanzo
-  - swiftdiaries
   - thesuperzapper
+  - yanniszark
 reviewers:
-  - SachinVarghese
\ No newline at end of file
+  - SachinVarghese
```

Comment:
@StefanoFioravanzo is there any reason you left this reviewer?

Reply:
It would be great if you could add me to approvers as well. Thanks

Reply:
@SachinVarghese, for now only leads of the Notebook WG can be approvers.